### PR TITLE
release notes 10.6.7

### PR DESCRIPTION
**Pull request by swethakundaram**

### CODE REVIEW

File: learn/wavemaker-release-notes/v10-6-7.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+## Bug Fixes
+
+---
+
+- We fixed an issue related to logout action failing with NPE (Null Pointer Exception) when using SSO OpenID with Horizontal Scaling enabled.
+- We fixed an issue related to the **Match Mode** property in the Data table form, forming an altered FilterExperssion for autocomplete, select, and search type.
+- We fixed an issue related to values modifying from negative to positive automatically when specified in the Number widget.
+- We fixed an issue related to the browser's autocomplete suggestion even when the **Autocomplete** property is turned off.
+- We fixed an issue related to the **On Change** event of the Number widget. Precisely, when a number widget is inside a Live form, it automatically triggers the **On Change** event when there are no actual changes in the event value.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-6-7.md:23]: "Precisely, when a number widget was inside a Live form, it used to automatically trigger the **On Change** event even when there were no actual changes in the number value."

File: learn/wavemaker-release-notes/v10-6-7.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+- We disabled the smooth-scroll by default and enabled the native scroll.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-6-7.md:24]: If it is specific to mobile, please mention it.

File: learn/wavemaker-release-notes/v10-6-7.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+- We fixed an issue related to duplicate Chip widgets adding-up when pressing the Enter key in preview and deploy modes.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-6-7.md:25]: No need to mention preview/deploy modes

**Review by vibhus — CHANGES_REQUESTED** [review]

File: learn/wavemaker-release-notes/v10-6-7.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+id: ""
+sidebar_label: "Release v10.6.7"
+---
+*WaveMaker released a new update version with some bug fixes.*
+
+## Improvements
+
+---
+
+- We introduced a new property called **Show Display Format On** for the Text widget to apply a defined **Display Format** property when entering text.
```
[the remark this answers is not in the document]
Comment (swethakundaram) [reply learn/wavemaker-release-notes/v10-6-7.md]: @swathianupuram will add the documentation, and we can link to it when it is available.

File: learn/wavemaker-release-notes/v10-6-7.md (hunk 2 above)
Comment (swethakundaram) [reply learn/wavemaker-release-notes/v10-6-7.md]: Done.

File: learn/wavemaker-release-notes/v10-6-7.md (hunk 3 above)
Comment (swethakundaram) [reply learn/wavemaker-release-notes/v10-6-7.md]: Okay.

**Review by vibhus — APPROVED** [review]

**Pull request merged**